### PR TITLE
Update for 2.2 release

### PR DESCRIPTION
**Pull request by mightybyte**

### CODE REVIEW

File: node/ChainwebNode.hs [hunk 1]
```diff
@@ -496,7 +496,7 @@ pkgInfoScopes =
 -- KILLSWITCH for version 2.1
```
Comment (larskuhtz) [inline node/ChainwebNode.hs:496]:
```suggestion
-- KILLSWITCH for version 2.2
```

**Review by larskuhtz — COMMENTED** [review]
I think, there's also some place in the Rosetta tests that uses the current version as a constant.

**Comment by mightybyte** [comment]
> I think, there's also some place in the Rosetta tests that uses the current version as a constant.

@LindaOrtega ?

**Comment by LindaOrtega** [comment]
> > I think, there's also some place in the Rosetta tests that uses the current version as a constant.
> 
> @LindaOrtega ?

I think this is the place that Lars is referring to, but it seems to read the version from a variable called `VERSION_chainweb`: https://github.com/kadena-io/chainweb-node/blob/master/test/Chainweb/Test/Rosetta/RestAPI.hs#L394

This is the PR that first made use of that variable: https://github.com/kadena-io/chainweb-node/pull/1142/files

**Review by larskuhtz — APPROVED** [review]

**Pull request merged**